src/main/resources/openapi.yml

```diff
@@ -1720,6 +1720,39 @@ paths:
               schema:
                 $ref: '#/components/schemas/ProblemDetail'
   /users:
+    get:
```

Comment:
Lets not add additional endpoints if in original endpoint it was not mentioned.

Reply:
I do not agree.

> Analyze and fix the N+1 select problem in the following entity relationships within the PetClinic REST API:

Task description states that we are working withing REST api. Let's just not test relationships that are not covered by api.